### PR TITLE
feat(notifications): Added email notifications also prettier.

### LINKED ISSUE #583: [FEATURE REQ] Email as a notifier, or perhaps Shoutrrr
hi! thanks for a very nice project. currently you support Telegram, Discord, and Notifiarr

I don't use any of these currently, but I do use email. are there plans to support it / SMTP? if so I'd be happy help add it myself

------------

alternatively there is a go library called Shoutrrr (inspired by [apprise](https://github.com/caronc/apprise))

https://containrrr.dev/shoutrrr/v0.5/

which supports Telegram, Discord, and many more (like email)

users just provide their own simple notification URIs like `discord://token@channel`

### PR DESCRIPTION
**Pull request by KaiserBh**
This should close #583, I tested it using google SMTP with their TLS and SSL it seem to be working but additional testing will always be appreciated. Also local test using Mailhog which is working fine.

Added prettier as well that way we have some sort of formatting, feel free change the prettier.rc config.

### CODE REVIEW

File: web/.eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          "argsIgnorePattern": "^_",
+          "caughtErrorsIgnorePattern": "^_",
+          "ignoreRestSiblings": true
+        }],
+        // Turn this on when we are ready for stricter type checking
+        "@typescript-eslint/no-explicit-any": "off",
+      },
+    },
+  ]
+};
```
Comment (fabricionaweb) [inline web/.eslintrc.cjs:60]: its missing EOF here

**Comment by fabricionaweb** [comment]
We may want to discuss about the indent for switch cases

**Comment by sam272** [comment]
@zze0s any timeline for getting this PR in a release?

**Comment by zze0s** [comment]
> @zze0s any timeline for getting this PR in a release?

Not ATM, slowly working my way through the open PRs.

**Comment by KaiserBh** [comment]
Thank you for your dedication and the effort you have put into this pull request. After a thorough review, we have decided to close this PR for now, as it has become a bit dated in the context of our current codebase. However, we recognize the value in the ideas and implementations you've proposed.

We appreciate the work done, especially the inclusion of features like Prettier. It would be beneficial for future collaboration if similar changes are submitted as separate PRs, to streamline the review process and integrate smoothly with our evolving codebase.

This is not the end of this journey. We encourage you to revisit these concepts in the future when they align more closely with our project's needs and direction. Please feel free to reach out with any new ideas or updates.

Thank you once again for your contribution and understanding.

regards,
z (jks kaiser)

**Pull request closed without merging**